content/customize-gnome.md

```diff
@@ -18,117 +18,141 @@ section: software
+The GNOME desktop environment is highly customizable. Visit [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed extensions.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
 
-To install a customization, simply head to [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed customizations.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
-
-We will share some of the more popular extensions, but there are many more to choose from!  If one is found and it isn't currently working, or breaks with an update, then try a different extension to see if it works better.
+**NOTE:** There are many extensions to choose from! If one isn't currently working, or breaks with an update, then try a different extension to see if it works better.
 
 ---
 
-## User Themes
+## Included Extensions
 
-[User Themes](https://extensions.gnome.org/extension/19/user-themes/)
+Pop!\_OS includes the GNOME Extensions app by default. Which lists both system-installed, and user-installed extensions.
 
-This extension allows for the desktop to be themed in further ways than default.  Many themes and extensions can be installed from [opendesktop.org](https://www.opendesktop.org/s/Gnome), among others.  This extension is installed by default in Pop!_OS, and can be added to Ubuntu 18.04.  
+![extensions-app-window](/images/customize-gnome/extensions-app-window.png)
 
----
+Each extension has options for customizing settings (if supported in Extension metadata), or viewing the webpage for the extension to learn more information. The Extensions app also allows you to quickly enable and disable installed extensions.
 
-## Dash to Dock
+Pop!\_OS includes the following extensions by default. In most cases, these extensions should be left **enabled**.
 
-[Dash to Dock](https://extensions.gnome.org/extension/307/dash-to-dock/)
+### [Cosmic Dock](https://github.com/pop-os/cosmic-dock)
 
-![dash-to-dock](/images/customize-gnome/dash-to-dock.png)
+A fork of Dash to Dock customized for Pop!\_OS.
 
-This extension brings a launcher similar to the Unity launcher on the left side, or the macOS launcher on the bottom.  It is high customizable and allows for different locations, styles, sizes, and layouts.  This is included by default in Ubuntu 18.04 and can be added to Pop!_OS.
+![cosmic-dock-close](/images/customize-gnome/cosmic-dock-close.png)
 
----
+**NOTE:** It is **not** recommended to have both Cosmic Dock and Dash-t0-Dock installed an enabled at the same time. User Interface (UI) bugs will occur. Choose one or the other.
```

Comment:
"Dash-t0-Dock" should be "Dash-**to**-Dock" and "installed an enabled" should be "installed **and** enabled".

Also, might be worth recommending COSMIC dock, as it comes pre-included with Pop!_OS and serves largely the same purpose as Dash-to-Dock does.

Reply:
Thanks for catching that typo. We'll fix that.

COSMIC Dock is a fork of Dash to Dock, but users can choose to use either one if they prefer a more standard GNOME experience.